### ISSUE TITLE
[BUG] ERR_NAME_NOT_RESOLVED when logging in via Microsoft

### ISSUE THREAD

**Issue by Jenipherr** [QuestCraftPlusPlus/QuestCraft#7]
**Describe the bug**
Unable to login via Microsoft due to an error screen after clicking "Microsoft Login"

**To Reproduce**
1. Open the app
2. Click "Microsoft Login"

**Expected behavior**
It's expected that a login page would appear; however, I'm just met with an error screen with the code "ERR_NAME_NOT_RESOLVED"

**Screenshots**

![com oculus shellenv-20220221-175839](https://user-images.githubusercontent.com/34631515/155042042-0b26ca32-dd4b-4cb8-8ff0-cd5f8fdf09ad.jpg)

**Headset Details (please complete the following information):**
 - Quest Build: Latest Oculus version
 - Device: Quest 1
 - QuestCraft Version: Latest QuestCraft version uploaded to Discord

**Comment by ManiacGamer156** [comment]
That will only happen if your headset is not connected to Wi-Fi, or is is blocked by DNS.

**Comment by Jenipherr** [comment]
> That will only happen if your headset is not connected to Wi-Fi, or is is blocked by DNS.

I'm connected to wifi and it shouldn't be blocked by DNS as I'm able to login via Microsoft oauth on my computer while connected to the same network.

**Comment by ManiacGamer156** [comment]
That is quite odd, but all the staff cannot reproduce that…

**Comment by Jenipherr** [comment]
> That is quite odd, but all the staff cannot reproduce that…

I'm not sure what to do about it then; I've tried using two other networks, and have reinstalled twice. All to no avail.

**Comment by Exist43** [comment]
this is sussy.
I'm having this problemo to, and I don't know how to fix it like I do the wiring in electrical.

I don't want to vent, I'm the sussy among us imposter please don't vote me out, and please someone find a solution sussy bakas

**Comment by Exist43** [comment]
when the imposter is sus! when the imposter is sus! when the imposter is sus! when the imposter is sus! when the imposter is sus! when the imposter is sus! when the imposter is sus! when the imposter is sus! when the imposter is sus! when the imposter is sus!

**Comment by ItsCoal** [comment]
im having this problem as well and there seems to be no way for me to fix it so idk thinking of just giving up

**Comment by ItsCoal** [comment]
So I have found a fix after reinstalling multiple times to lead to zero fixes. I tried something different even though my internet was perfect and there was nothing wrong with it. I decided to make a hotspot on my phone and connect to it via the quest 2 now after connecting to the hotspot I did the usual check keep me logged in and chose Microsoft login and to my surprise it loaded the login page right up I went entered my account info and signed in afterwards I disconnected from my hotspot and reconnected to my original connection and all was good I was happily playing QuestCraft.
